### ISSUE TITLE
1.2.3 release tag missing

### ISSUE THREAD

**Issue by glensc** [Linuxbrew/brew#422]
https://github.com/Linuxbrew/brew/issues/1#issuecomment-313190533 released 1.2.3 (can't comment there), but the git tag has not been [pushed](https://github.com/Linuxbrew/brew/releases)

```
➔ brew update
Already up-to-date.
➔ brew --version
Homebrew 1.2.2-69-g286d66673
Homebrew/homebrew-core (git revision f310d8; last commit 2017-07-05)
➔ cd `brew --repo`
➔ git fetch --tags
➔ git describe --tags
1.2.2-69-g286d66673
➔
```

/cc @iMichka

**Comment by glensc** [comment]
suggested fix:


- git tag 1.2.3 29ed987a7
- git push origin 1.2.3


or 
- git tag 1.2.3 b26db51d0
- git push origin 1.2.3

depending where you want to add the tag

**Comment by iMichka** [comment]
Hi. Thanks for reporting. It's my fault. I was too eager to make the announcement. We normally wait 1-2 days before announcing the release. The merge has been done. Don't worry, I'll push the tag tomorrow.

**Comment by glensc** [comment]
oh, awesome, i hope my change from #417 get included in 1.2.3, useful for new installations.

**Comment by sjackman** [comment]
Yes, your PR #417 will be included in Linuxbrew 1.2.3.
